client/landing/stepper/declarative-flow/onboarding.ts

```diff
@@ -365,11 +398,16 @@ const onboarding: Flow = {
 						return navigate( 'designSetup' );
 					}
 				case 'designSetup':
+					if ( isDesignChoicesStepEnabled ) {
+						return navigate( 'design-choices' );
+					}
 					if ( isGoalsAtFrontExperiment ) {
 						return navigate( 'goals' );
 					}
 				case 'difmStartingPoint':
 					return navigate( 'goals' );
+				case 'design-choices':
```

Comment:
The beauty of the stepper.

Reply:
That's why it was possible to slap all these existing pieces together so quickly!